### PR TITLE
Wip new

### PR DESCRIPTION
**Pull request by joshuabamboo**
@AnnJohn @jmburges

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+Let's go through an example to clarify remote repositories on Github.
+
+##Create a local directory
+1. Create a folder: `mkdir git-remote-code-along`
+2. Add a readme: `touch READme.md`
+
+##Creating a remote repository on Github
+
+1. From your Github profile, click the "Create New Repository" button.
+2. After you create the repo, click the "Copy to clipboard" symbol on the right-hand side of the screen to copy the url (we'll use this in the next section)
```
Comment (jmburges) [inline README.md]: call it the "clone url"

File: README.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+
+
+##Connecting your remote repo to a local repo
+1. `git --init`
+2. `git add .` + `git commit -m "initialize git"`. Add and commit the new file created by the init command in the step 1.
+3. `git remote add origin your-remote-repository-URL`. This sets the remote, so you can push and pull code.
+
+##Push code to Github
+1. Let's add something to our READme. Open the file and add whatever text you'd like.
```
Comment (jmburges) [inline README.md]: README caps!

File: README.md [hunk 3]
```diff
@@ -1 +1,50 @@
...
+Let's say you liked your READme, but you noticed a minor typo. Let's fix it directly on Github by clicking the READme.md file. 
+
+1. At the top of the file, you'll notice a pencil icon. Clicking this will allow us to edit the file.
+2. Make some changes to the READme
+3. Commit them by clicking the "Commit changes" button at the bottom of the page.
+
+Perfect! The only problem is, now our code on our machine (our local repo) is out of sync with the remote repo. To remedy this, we must pull down the code to our local repo. No surprises here. To do this we run:
+
+```bash
+git pull
```
Comment (jmburges) [inline README.md:55]: do a `git pull` before you make an update to show what it looks like when there is nothing to pull

**Comment by joshuabamboo** [comment]
All done. :shipit: ?? @jmburges

**Comment by jmburges** [comment]
:shipit:

**Pull request merged**